### PR TITLE
Feature/create wig and bed index

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- See above

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Tiddit_coverage.pm [hunk 1]
```diff
@@ -248,6 +276,17 @@ sub analysis_tiddit_coverage {
             }
         );
 
+        set_file_path_to_store(
+            {
+                format           => q{wig},
+                id               => $sample_id,
+                path             => $outfile_path,
+                path_index       => $index_file_path,
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Tiddit_coverage.pm]: can't we just store the bigwig file or are there any downstream programs that will use the wig?

File: lib/MIP/Recipes/Analysis/Rhocall.pm [hunk 2]
```diff
@@ -585,13 +611,24 @@ sub analysis_rhocall_viz {
...
                 sample_info_href => $sample_info_href,
             }
         );
 
+        set_file_path_to_store(
+            {
+                format           => q{wig},
+                id               => $sample_id,
+                path             => $viz_wig_outfile_path_prefix . $DOT . q{wig},
+                path_index       => $viz_wig_outfile_path_prefix . $DOT . q{bw},
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Rhocall.pm]: I thought that we only needed the bigwig file?

File: lib/MIP/Sample_info.pm [hunk 3]
```diff
@@ -660,7 +660,7 @@ sub set_file_path_to_store {
 
     my $tmpl = {
         format => {
-            allow       => [qw{ fastq bam bcf cram meta vcf }],
+            allow       => [qw{ fastq bam bcf bed cram meta tar vcf wig }],
```
Comment (jemten) [inline lib/MIP/Sample_info.pm]:
```suggestion
            allow       => [qw{ bam bcf bed cram fastq meta tar vcf wig }],
```

File: t/ucsc_bed_to_big_bed.t [hunk 4]
```diff
@@ -25,7 +25,7 @@ use MIP::Test::Commands qw{ test_function };
 use MIP::Test::Fixtures qw{ test_standard_cli };
 
 my $VERBOSE = 1;
-our $VERSION = 1.00;
+our $VERSION = 1.01;
```
Comment (jemten) [inline t/ucsc_bed_to_big_bed.t]:
```suggestion
our $VERSION = 1.00;
```

File: t/ucsc_wig_to_big_wig.t [hunk 5]
```diff
@@ -99,6 +99,10 @@ my %required_argument = (
 );
 
 my %specific_argument = (
+clip => {
+    input           => 1,
+    expected_output => q{-clip},
+},
```
Comment (jemten) [inline t/ucsc_wig_to_big_wig.t]: indent

File: lib/MIP/Recipes/Analysis/Upd.pm [hunk 6]
```diff
@@ -268,6 +298,17 @@ sub analysis_upd {
                 sample_info_href => $sample_info_href,
             }
         );
+        my $index_file_path_prefix = fileparse( $outfile_path{sites}, qr/[.]bed/sxm );
+        set_file_path_to_store(
+            {
+                format           => q{bed},
+                id               => $sample_id,
+                path             => $outfile_path{sites},
+                path_index       => $index_file_path_prefix . $DOT . q{bb},
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Upd.pm]: same as above

File: lib/MIP/Recipes/Analysis/Tiddit_coverage.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Tiddit_coverage.pm]: Not that I know of. Felt like they belong together, but I think you are right. Let's store only the binary file.

File: lib/MIP/Recipes/Analysis/Rhocall.pm (hunk 2 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Rhocall.pm]: I will change it to only store the bigwig

File: lib/MIP/Recipes/Analysis/Tiddit_coverage.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Tiddit_coverage.pm]: Done

File: lib/MIP/Recipes/Analysis/Rhocall.pm (hunk 2 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Rhocall.pm]: Done

File: t/ucsc_wig_to_big_wig.t (hunk 5 above)
Comment (henrikstranneheim) [reply t/ucsc_wig_to_big_wig.t]: Fixed

File: lib/MIP/Recipes/Analysis/Upd.pm (hunk 6 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Upd.pm]: Done

**Pull request merged**